### PR TITLE
[camera] Ios support documentation 

### PR DESCRIPTION
**Pull request by danielroek**
## Description

Updated the README about the minimum iOS version as proposed [here](https://github.com/flutter/plugins/pull/2753). 

## Related Issues

PR #2753 should be closed after merging this PR.  

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/camera/camera/README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.
 
 ### iOS
 
+iOS 10.0 of higher is needed to use the camera plugin. If compiling for any version lower than 10.0 make sure to check the iOS version with for example the [device_info plugin](https://pub.dev/packages/device_info)
```
Comment (mvanbeusekom) [inline packages/camera/camera/README.md:22]: nit: I think the explanation can be improved a little bit by converting the sentence to:

```suggestion
iOS 10.0 of higher is needed to use the camera plugin. If compiling for any version lower than 10.0 make sure to check the iOS version before using the camera plugin. For example, using the [device_info](https://pub.dev/packages/device_info) plugin.
```

**Review by mvanbeusekom — COMMENTED** [review]
LGTM, have one small nit regarding the sentence. Would be nice if you can have a look at it.

**Review by mvanbeusekom — APPROVED** [review]
LGTM

**Review by amirh — APPROVED** [review]
LGTM

**Pull request merged**